packages/foreman/foreman/foreman.spec

```diff
@@ -24,6 +24,7 @@ BuildArch:  noarch
 
 # Plugin was removed in Foreman 3.3, 3.5 includes DB cleanup
 Obsoletes: rubygem-foreman_docker < 5.0.0-4
+Obsoletes: %{name}-ovirt < %{version}-%{release}
```

Comment:
Can you also add a comment that this was done in 3.16 so in the future we know roughly how old it was?